Casks/a/attachecase.rb

```diff
@@ -10,10 +10,13 @@
 
   livecheck do
     url "https://hibara.org/software/attachecase/json/attachecase_mac_history_en.json"
-    regex(/AtcMacv?(\d)(\d)(\d)\.zip/i)
-    strategy :page_match do |page, regex|
-      page.scan(regex).map do |match|
-        "#{match[0]}.#{match[1]}.#{match[2]}"
+    regex(/^ver\.(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^\D*(\d+(?:\.\d+)+)$/i)
```

This regex would fail to match a future `version` with a different prefix (e.g., `v.`, `ver`, `v`, etc.) or no prefix at all. We could use something more flexible like `(?:v(?:er)?\.?)?` but it should be okay to simply use `\D*` in this context (e.g., like we do in the `Git` strategy's default regex) or not anchor the start of the regex (`(\d+(?:\.\d+)+)$/i`).

Reply:
Thanks, that makes sense!